chat-android/src/main/java/com/ably/chat/Typing.kt

```diff
@@ -57,21 +44,21 @@ public interface Typing : EmitsDiscontinuities {
     public fun subscribe(listener: Listener): Subscription
 
     /**
-     * Get the current typers, a set of clientIds.
+     * Current typing members.
      * @return set of clientIds that are currently typing.
      */
-    public suspend fun get(): Set<String>
+    public fun get(): Set<String>
 
     /**
-     * Start indicates that the current user is typing. This will emit a typingStarted event to inform listening clients and begin a timer,
-     * once the timer expires, a typingStopped event will be emitted. The timeout is configurable through the typingTimeoutMs parameter.
-     * If the current user is already typing, it will reset the timer and being counting down again without emitting a new event.
+     * Keystroke emits a typingStarted event to inform listening clients.
```

Comment:
We could probably add a bit more detail here - worth copying the comment for this and `stop` from JS?

Reply:
Sure, will update accordingly 👍 
We are creating new PR with proper commit history, will add you there 👍